src/opencensus/proto/trace/v1/trace.proto

```diff
@@ -26,7 +26,7 @@ option java_outer_classname = "TraceProto";
 option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1";
 
 // A span represents a single operation within a trace. Spans can be
-// nested to form a trace tree. Often, a trace contains a root span
+// nested to form a trace graph. Often, a trace contains a root span
```

Comment:
Please revert this, "graph" is the general case and allows cycles i.e. it would imply that span A could be the parent of span B and span B could the parent of span A; tree is a directed acyclic graph so the wording was correct before, unless I missed something /cc @bogdandrutu

Reply:
`Links` make trace a graph, not a tree. Without `Links` it will be a tree

Reply:
Right, but currently the comment just mentions spans and not links, of which your change perhaps assumes that the user knows about links and I am not sure that this was meant to introduce links so far.